dir.props

```diff
@@ -148,9 +154,6 @@
     <ConfigurationGroup Condition="'$(ConfigurationGroup)' == ''">Debug</ConfigurationGroup>
     <ArchGroup Condition="'$(ArchGroup)' == ''">x64</ArchGroup>
 
-    <!-- For VS support if Configuration is set we default BuildConfiguration based on that property -->
-    <BuildConfiguration Condition="'$(BuildConfiguration)' == '' and '$(Configuration)' != ''">$(Configuration)-$(ArchGroup)</BuildConfiguration>
```

Comment:
Just for reference, this was done because we support running only netcoreapp currently and have no way of  configuring a VS project to run for other implementations.